python/lsst/ts/salobj/configurable_csc.py

```diff
@@ -372,13 +370,11 @@ async def begin_start(self, data):
                 githash = self.evt_settingVersions.data.recommendedSettingsVersion
                 config_file_name = self.config_label_dict.get(config_name, config_name)
                 config_file_path = self.config_dir / config_file_name
-                if not config_file_path.is_file():
-                    raise base.ExpectedError(
-                        f"Cannot find config file {config_file_name} "
-                        f"in {self.config_dir}"
-                    )
-                with open(config_file_path, "r") as f:
-                    config_yaml = f.read()
+                try:
+                    with open(config_file_path, "r") as f:
+                        config_yaml = f.read()
+                except OSError as e:
+                    raise base.ExpectedError(f"Cannot read {config_file_path}: {e}")
```

Comment:
Please provide context by saying what the file is that it can't read, e.g.: `f"Cannot read config file {config_file_path}: {e}"`